src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.types.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    IColumn,
+    IStyle,
+    IStyleFunctionOrObject,
+    ITheme
+} from '@fluentui/react';
+import {
+    IADTTwin,
+    IPropertyInspectorAdapter,
+    Locale,
+    Theme
+} from '../../../../Models/Constants';
+
+export interface IAdvancedSearchResultDetailsListProps {
+    filteredTwinsResult: IADTTwin[];
+    additionalColumns: IColumn[];
+    // callback function from parent on what to do once the user selects or deselects a twin
+    callbackFunction?: void; //not meant to be optional, just did that for now so the stories work
```

Comment:
let's call this `onSelectionChange` and it should take the IADTTwin for that selected row as an argument